iocore/net/UnixNet.cc

```diff
@@ -507,27 +507,24 @@ NetHandler::waitForActivity(ink_hrtime timeout)
       if (cop_list.in(ne)) {
         cop_list.remove(ne);
       }
-      if (get_ev_events(pd, x) & (EVENTIO_READ | EVENTIO_ERROR)) {
+      int flags = get_ev_events(pd, x);
```

Comment:
Is it possible to get only error bits and not read or write ready bits? It's reasonable to presume there's no `epoll` result unless checking for one of those conditions, so it's possible the checked for condition is always present.

Reply:
There is a ink_release_assert later in this function for that case.  I have yet to see it go off.